README.rst

```diff
@@ -206,6 +206,10 @@ tt    Time                                        time
       e.g. 10:21:36 PM -5:30
 ===== =========================================== ========
 
+The type can also be a datetime format string, following the
+`1989 C standard format codes`_, e.g. %Y-%m-%d. Any type containing %Y
+or %y will be parsed and output as a ``datetime.datetime``.
```

Comment:
The condition on %Y or %y seems strange to me, but I don't have a better idea (since the strftime string can be just about anything...)

Reply:
Yes but I couldn't think of a case where you wouldn't use a year. Technically if you don't provide one it does parse without error, defaulting to using 1900 as the year, but I didn't feel like I wanted to support that edge case

Reply:
Parsing times comes to mind, e.g. `"train departs the station at %H:%M"`

Reply:
Yes, but what would it return? As far as I know, Python does not have a native `time` class, so you end up with

```python
datetime.strptime("13:23", "%H:%M")
```

```python
datetime.datetime(1900, 1, 1, 13, 23)
```

which is probably not what the user intended

Reply:
ok, I take it back. It wasn't too bad to support time parsing